lectures/newtons_method.md

```diff
@@ -0,0 +1,469 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+ ```{raw} html
+ <div id="qe-notebook-header" align="right" style="text-align:right;">
+         <a href="https://quantecon.org/" title="quantecon.org">
+                 <img style="width:250px;display:inline;" width="250px" src="https://assets.quantecon.org/img/qe-menubar-logo.svg" alt="QuantEcon">
+         </a>
+ </div>
+ ```
+
+# Using Newton’s Method to Solve Economic Models
+
+In addition to what’s in Anaconda, this lecture will need the following libraries:
+```{code-cell} ipython3
+---
+tags: [hide-output]
+---
+!pip install quantecon
+```
+
+## Overview
+
+Continuing from the the [Newton's Method lecutre](https://python.quantecon.org/newton_method.html), we are going to solve the multidimensional problem with `Jax`. More information about Jax can be found [here](https://python-programming.quantecon.org/jax_intro.html).
+
+We use the following imports in this lecture
+
+```{code-cell} python3
+import jax
+import jax.numpy as jnp
+import quantecon as qe
+import numpy as np
+from scipy.optimize import root
+```
+
+## The Two Goods Market Equilibrium
+
+Let's have a quick recall for this problem, a more detailed explanation and derivation can be found at [A Two Goods Market Equilibrium](https://python.quantecon.org/newton_method.html#two_goods_market).
+
+Assuming we have such market for two complementary goods where demand depends on the price of both components. 
+
+We label them $\text{good}_0$ and $\text{ good}_1$, with price vector $p = (p_0, p_1)$.
+
+Then the supply of good $i$ at price $p$ is,
+
+$$ 
+q^s_i (p) = b_i \sqrt{p_i} 
+$$
+
+and the demand of good $i$ at price $p$ is,
+
+$$ 
+q^d_i (p) = \text{exp}(-(a_{i0} p_0 + a_{i1} p_1)) + c_i
+$$
+
+Here $a_{ij}$, $b_i$ and $c_i$ are parameters for $n \times n$ square matrix $A$ and $n \times 1$ parameter vectors $b$ and $c$.
+
+The excess demand function is,
+
+$$
+e_i(p) = q_i^d(p) - q_i^s(p), \quad i = 0, 1
+$$
+
+An equilibrium price vector $p^*$ satisfies $e_i(p^*) = 0$.
+
+We set
+
+$$
+A = \begin{pmatrix}
+            a_{00} & a_{01} \\
+            a_{10} & a_{11}
+        \end{pmatrix},
+            \qquad 
+    b = \begin{pmatrix}
+            b_0 \\
+            b_1
+        \end{pmatrix}
+    \qquad \text{and} \qquad
+    c = \begin{pmatrix}
+            c_0 \\
+            c_1
+        \end{pmatrix}
+$$
+
+for this particular question.
+
+
+### The Multivariable Market Equilibrium
+
+We can now easily get the multivariable version of the problem above.
+
+The supply function remain unchanged,
+
+$$ 
+q^s (\bf{p}) =\bf{b} \bf\sqrt{p} 
+$$
+
+The demand function is,
+
+$$ 
+q^d (\bf{p}) = \text{exp}(- \bf{A} \cdot \bf{p}) + \bf{c}
```

Comment:
This looks nice but actually our convention is we usually don't use bold for vectors and matrices --- sorry @HengchengZhang 

Please drop the `\bf`